### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by waqaskanju**
Add required attribute to contact form
Add action attribute to contact form
Add submit attribute to contact form

### CODE REVIEW

File: work.html [hunk 1]
```diff
@@ -225,42 +225,53 @@ <h3 class="header-3">Skills</h3>
...
+                name="textarea"
+                id="textarea"
+                cols="30"
+                rows="8"
+                placeholder="Write your message here"
+                maxlength="500"
+                required
+              ></textarea>
+              <input class="contact_submit" type="submit" value="Submit" />
+            </form>
```
Comment (zieeco) [inline work.html:271]: - [ ] **[REQUIRED CHANGES]**

> || **YOUR DESIGN**|| **FIGMA DESIGN** ||
> |-|---------|-|---------|-|
> || space || space2 || 

- Please, kindly remove the excess space, add the line and change the text content of your button to match that of the Figma design. 

- Please update your readme by doing the following:

1. change your project title
2. Remove the `author header` after the `Built with` header and `Project Name` header.
3. Add the second author that contributed to this project.
4. Remove the `Deployment`, `Usage`, `Run test`, `Install`, and `setup` as they are not descriptive enough.
5. Update your `Prerequisite`  e.g 
```

## Getting Started

To get a local copy up and running follow these simple example steps.

### Getting started
 - A text editor(preferably Visual Studio Code)

### Install
  -  [Git](https://git-scm.com/downloads)
### Usage
#### Clone this repository

```bash
$  https://github.com/waqaskanju/Portfolio.git
$ cd Portfolio
```
```

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @waqaskanju @darikmohammed 


Great work on your contact form 👏 
You've done well implementing its features, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A
**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: work.html (hunk 1 above)
Comment (waqaskanju) [reply work.html]: Thanks for your suggestions. Form and read me file updated as per your instructions.

**Review by koomekevo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: 
Hi @waqaskanju , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: You have implemented changes suggested by a previous reviewer. 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**